src/SmartFormat/Utilities/PluralRules.cs

```diff
@@ -284,17 +298,20 @@ value switch
             _ => 5  // other
         };
     private static PluralRuleDelegate Manx => (value, pluralWordsCount) =>
-        (value % 10).Between(1, 2) || value % 20 == 0
+        (value % 10).BetweenWithoutFraction(1, 2) || value % 20 == 0
             ? 0
             : // one
             1;
     private static PluralRuleDelegate Langi => (value, pluralWordsCount) =>
-        value == 0 ? 0 : // zero
-        value is > 0 and < 2 ? 1 : // one
-        2;
+        value switch
+        {
+            0 => 0,
```

Comment:
Do we need to handle negative values?

`<=0 => 0`

Reply:
Good point. Throwing for negative values would mean a breaking change.
With other rules, negative values return the maximum index. This is not handled consistently. What do you think?

Reply:
Is there a difference between a negative plural and a positive in any language? Maybe we just need to Abs the value and treat it as positive before we evaluate it?

Reply:
Hm, not 100% sure for each language. As I see it, it's not about the language, but rather about the implementation. The Abs solution you mentioned could be a way to better consistency.